src/detection/card/items/camera/hooks/use-on-click.js

```diff
@@ -0,0 +1,4 @@
+import { useNavigation } from 'detections/table/hooks/use-navigation'
+import { useItem } from 'detection/card/use-item'
+
+export const useOnClick = () => ({ onClick: useNavigation(`/cameras?id=${useItem().camera.id}`) })
```

Comment:
Why don't we return a `useCallback` from this hook? It will make its usage clearer and, more importantly, safer.

Reply:
Done